contracts/Curation.sol

```diff
@@ -0,0 +1,529 @@
+pragma solidity 0.4.18;
+
+import "@aragon/os/contracts/apps/AragonApp.sol";
+import "@aragon/os/contracts/lib/misc/Migrations.sol";
+import "@aragon/os/contracts/lib/zeppelin/math/SafeMath.sol";
+import "@aragon/os/contracts/lib/zeppelin/math/SafeMath64.sol";
+
+import "@aragon/apps-staking/contracts/interfaces/IStaking.sol";
+import "@aragon/apps-registry/contracts/interfaces/IRegistry.sol";
+import "./interfaces/IVoting.sol";
+
+
+contract Curation is AragonApp {
```

Comment:
We should consider all the cases in which locks are created to the curation app that may result in tokens getting locked forever, for example, locking tokens to create an application or challenge and being frontrun in the action. We need a mechanism to unlock when the locks haven't been used.

Comment on the topic for PLCR: https://github.com/aragonlabs/plcr/pull/1/files#r206916590

Reply:
Yes, another related example: https://github.com/aragonlabs/curation/issues/16